### DISCUSSION TITLE
Physics process for 3D

### DISCUSSION THREAD

**Discussion by yeet227** [godotengine/godot-proposals#5080] [Editor]
For 3D meshes could there be a node or a mesh editor that could change mass, weight, whether the mesh has gravity similar to Unreal Engine without Code.![image](https://user-images.githubusercontent.com/51895536/183109738-1881ffe4-cf95-4448-b2c1-b58de66dd5b3.jpeg)

**Comment by Lielay9** [comment]
Are there some restrictions that make it difficult or impossible to use a [rigidbody](https://docs.godotengine.org/en/stable/classes/class_rigidbody.html)? Wouldn't you need to use one anyway if the mesh is intended to interact with physics?

**Reply by yeet227** [reply]
Thanks for the advice I just realised that you could do that I was expecting more of a Unreal engine like experience when making interactive objects like that
